indigo/analysis/refs/base.py

```diff
@@ -63,6 +64,51 @@ class RefsFinderENG(BaseRefsFinder):
     candidate_xpath = ".//text()[contains(., 'Act') and not(ancestor::a:ref)]"
 
 
+@plugins.register('refs-subtypes')
+class RefsFinderSubtypesENG(BaseRefsFinder):
+    """ Finds references to works other than Acts in documents, of the form:
+
+        P 52 of 2001
+        Ordinance no. 52 of 1998
+        GN 1/2009
+
+    """
+
+    # country, language, locality
+    locale = (None, 'eng', None)
+
+    subtypes = [s for s in Subtype.objects.all()]
```

Comment:
Don't access the database when declaring a class. This needs to be done in the `setup` method or similar.

This is probably the cause of the test failures.